--- wp-config-sample-ja.php	2025-10-15 14:23:22.720834724 +0900
+++ wp-config-custom-ja.php	2025-10-15 14:11:35.566264363 +0900
@@ -1,4 +1,10 @@
 <?php
+$ssl=false;
+define('MY_SITE','HTTPS_DOMAIN');
+$_SERVER['HTTP_HOST'] = MY_SITE;
+define('WP_HOME','http'.($ssl ? 's' : '').'://'.MY_SITE);
+define('WP_SITEURL','http'.($ssl ? 's' : '').'://'.MY_SITE);
+$_SERVER['HTTPS'] = $ssl ? 'on' : 'off';
 /**
  * WordPress の基本設定
  *
@@ -18,7 +24,7 @@
  * @package WordPress
  */
 
-// 注意: 
+// 注意:
 // Windows の "メモ帳" でこのファイルを編集しないでください !
 // 問題なく使えるテキストエディタ
 // (http://wpdocs.sourceforge.jp/Codex:%E8%AB%87%E8%A9%B1%E5%AE%A4 参照)
@@ -48,18 +54,18 @@
  *
  * それぞれを異なるユニーク (一意) な文字列に変更してください。
  * {@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org の秘密鍵サービス} で自動生成することもできます。
- * 後でいつでも変更して、既存のすべての cookie を無効にできます。これにより、すべてのユーザーを強制的に再ログインさせることになります。
+ * 後でいつでも変更して、既存のすべての cookie を無効にできます。これにより、すべてのユーザーを制的に再ログインさせることになります。
  *
  * @since 2.6.0
  */
-define('AUTH_KEY',         'put your unique phrase here');
-define('SECURE_AUTH_KEY',  'put your unique phrase here');
-define('LOGGED_IN_KEY',    'put your unique phrase here');
-define('NONCE_KEY',        'put your unique phrase here');
-define('AUTH_SALT',        'put your unique phrase here');
-define('SECURE_AUTH_SALT', 'put your unique phrase here');
-define('LOGGED_IN_SALT',   'put your unique phrase here');
-define('NONCE_SALT',       'put your unique phrase here');
+define( 'AUTH_KEY',         'put your unique phrase here' );
+define( 'SECURE_AUTH_KEY',  'put your unique phrase here' );
+define( 'LOGGED_IN_KEY',    'put your unique phrase here' );
+define( 'NONCE_KEY',        'put your unique phrase here' );
+define( 'AUTH_SALT',        'put your unique phrase here' );
+define( 'SECURE_AUTH_SALT', 'put your unique phrase here' );
+define( 'LOGGED_IN_SALT',   'put your unique phrase here' );
+define( 'NONCE_SALT',       'put your unique phrase here' );
 
 /**#@-*/
 
@@ -90,4 +96,6 @@
         define('ABSPATH', dirname(__FILE__) . '/');
 
 /** Sets up WordPress vars and included files. */
-require_once(ABSPATH . 'wp-settings.php');
+if ( ! defined( 'HEALTH_CHECK' ) ) {
+	require_once ABSPATH . 'wp-settings.php';
+}
\ No newline at end of file
