--- wp-config-sample.php	2025-12-23 11:30:49.811663935 +0900
+++ wp-config-custom.php	2025-12-23 16:25:17.804535735 +0900
@@ -1,4 +1,10 @@
 <?php
+$ssl=true;
+define('MY_SITE','HTTPS_DOMAIN');
+$_SERVER['HTTP_HOST'] = MY_SITE;
+define('WP_HOME','http'.($ssl ? 's' : '').'://'.MY_SITE);
+define('WP_SITEURL','http'.($ssl ? 's' : '').'://'.MY_SITE);
+$_SERVER['HTTPS'] = $ssl ? 'on' : 'off';
 /**
  * The base configuration for WordPress
  *
@@ -80,6 +86,7 @@
  * @link https://wordpress.org/support/article/debugging-in-wordpress/
  */
 define( 'WP_DEBUG', false );
+define( 'WP_CACHE', true );
 
 /* Add any custom values between this line and the "stop editing" line. */
 
@@ -93,4 +100,6 @@
 }
 
 /** Sets up WordPress vars and included files. */
-require_once ABSPATH . 'wp-settings.php';
+if ( ! defined( 'HEALTH_CHECK' ) ) {
+	require_once ABSPATH . 'wp-settings.php';
+}
